File: HeartDataMiniProgram/pages/monthReport/monthReport.js
// pages/monthReport/monthReport.js
Page({

  /**
   * 页面的初始数据
   */
  data: {
    year: '',
    month: '',
    monthReports: {}
  },

  /**
   * 生命周期函数--监听页面加载
   */
  onLoad: function (options) {
    var y = options.year;
    var m = options.month;
    var that = this;
    // 通过http请求，加载每年前两个月的报告
    
    // 解析JSON对象
    var res = '{ "data" : ['+
                '{ '+
                  '"year": "2020",'+
                  '"month": ['+
                    '{ "m": "1",' +
                      '"imgurl": "/static/imgs/reports/riqi1.png",'+
                      '"title": "1月报告",'+
                      '"description": "1月报告，良好",'+
                      '"analysis": {'+
                        '"HealthIndex": "8.0",'+
                        '"Other": ""'+
                      '},'+
                      '"dayLists":['+
                        '{ "d" : "1",'+
                          '"imgurl": "/static/imgs/reports/riqi1.png",'+
                          '"isUsed": "true",'+
                          '"HealthIndex" : "9.9"'+
                        '},'+
                        '{ "d" : "2",'+
                          '"imgurl": "/static/imgs/reports/riqi2.png",'+
                          '"isUsed": "false",'+
                          '"HealthIndex" : ""'+
                        '},'+
                        '{ "d" : "3",'+
                          '"imgurl": "/static/imgs/reports/riqi3.png",'+
                          '"isUsed": "true",'+
                          '"HealthIndex" : "1.5"'+
                        '}'+
                      ']'+
                    '}'+
                  ']'+
                '}'+
              ']}';
    var obj = JSON.parse(res);
    // console.log(obj);
    that.setData({
      year: y,
      month: m,
      monthReports: obj
    });
  }
})

File: HeartDataServer/src/main/java/com/qian/Entity/algorithm/MonthReport.java
package com.qian.Entity.algorithm;

import java.util.List;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class MonthReport {

	/*
	 * "m": "1",
       "imgurl": "/static/imgs/reports/riqi1.png",
       "title": "1月报告",
       "description": "1月报告，良好",
       "analysis": {
	       "HealthIndex": "8.0",
	       "Other": ""
       },
       "dayLists": [DayReport1, DayReport2...]
	 */
	/*
	 *   `id` int(11) NOT NULL AUTO_INCREMENT COMMENT '主键（自增长）',
		 `y` int(10) NOT NULL COMMENT '年份名称',
  		 `m` int(10) NOT NULL COMMENT '月份名称',
  		 `imgurl` varchar(100)  NOT NULL COMMENT '图片url',
		 `title` varchar(50) NOT NULL COMMENT '报告标题',
		 `description` varchar(100) NOT NULL COMMENT '简要描述',
		 `health_index` int(10) NOT NULL COMMENT '健康指标',
  		 `pepole_id` int(11) NOT NULL COMMENT '所属人Id',
	 */
	
	private int id;
	private int y;
	private int m;
	private String imgurl;
	private String title;
	private String description;
	// 分析
	public float healthIndex;
	public String other;
	
	private int pepoleId;
	private List<DayReport> dayLists;
}
